modules/es-extensions/publisher/extensions/app/greg_publisher/themes/default/pages/add-list-password.hbs

```diff
@@ -0,0 +1,194 @@
+    <link href="/publisher/themes/default/css/bootstrap-select.min.css" rel="stylesheet"/>
+    <link href="/publisher/extensions/app/greg-publisher-defaults/themes/default/css/select2.min.css" rel="stylesheet"/>
+    <link href="/publisher/themes/default/css/custom.css" rel="stylesheet">
+    <link href="/publisher/themes/default/css/custom-theme.css" rel="stylesheet">
+    <link href="/publisher/extensions/app/greg-publisher-defaults/themes/default/css/common.css" rel="stylesheet">
+    <!-- fontawesome -->
+    <link rel="stylesheet" href="/publisher/themes/default/css/font-awesome.min.css">
+    <!-- fontwso2 -->
+    <link rel="stylesheet" href="/publisher/themes/default/css/fontwso2.css">
+    <link rel="stylesheet" href="/publisher/themes/default/css/fontwso2-extend.css">
+
+    <link rel='stylesheet prefetch' href='http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/themes/smoothness/jquery-ui.css'>
+
+    <link rel="stylesheet" href="/publisher/themes/default/css/style.css">
+
+    <link rel="stylesheet" type="text/css" href="/publisher/themes/default/css/server-table.css">
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!--[if lt IE 9]>
+    <script src="/publisher/themes/default/js/html5shiv.min.js"></script>
+    <script src="/publisher/themes/default/js/respond.min.js"></script>
+    <![endif]-->
+    <title>WSO2 Governance Center - Publisher | WSO2 Enterprise Store back-office</title>
+
+</head>
+
+<body>
+<div class="wrapper">
+    <div class="container-fluid">
+        <!-- header -->
+        <header>
+            <div class="row wr-global-header">
+                <div class="col-xs-8 col-sm-6 col-lg-8 app-logo">
+                    <a href="/publisher/">
+                        <img src="/publisher/extensions/assets/server/themes/default/imgs/logo.png"/>
+
+                        <h2 class="app-title">Governance Center - Publisher</h2>
+                    </a>
+                </div>
+                <div class="col-xs-4 col-sm-6 col-lg-4 wr-auth-container">
+                    <div class="wr-auth pull-right">
+                        <a href="#" data-toggle="dropdown">
+                            <div class="auth-img">
+                                admin
+                           <span class="fw-stack-md fw-lg">
+                           <i class="fw fw-user fw-stack-1-5x"></i>
+                           </span>
+                            </div>
+                        </a>
+
+                        <div class="dropdown-menu">
+                            <div class="cu-arrow"></div>
+                            <div class="dropdown-menu-content">
+                                <a class="filter-item" onclick="window.location.href='/publisher/logout'">Sign out</a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </header>
+        <!-- /header -->
```

Comment:
Header is removed and, added this hbs as a partial. So the default header will be used.
PR #348